solutions/ceoi-17-PalindromicPartitions.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: ceoi-17-PalindromicPartitions
+source: CEOI
+title: 2017 - Palindromic Partitions
+author: Chongtian Ma
+---
+
+[Official Analysis](https://csacademy.com/contest/ceoi-2017-day-2/task/palindromic-partitions/solution/)
+
+## Implementation
+
+The greedy approach is implemented below. We will loop from left to right, adding characters one at a time until the left string is the same as the right string, in which we can create two partitions. To check the equality of these strings, we will use string hashing.
```

Comment:
adding characters to what?

Reply:
to both strings? what else?

Reply:
wdym "both strings"?
don't expect the reader to automatically know what variables you're using